### ISSUE TITLE
use return; when the function returns void values.

### ISSUE THREAD

**Issue by wickedOne** [djoos/Symfony-coding-standard#105]
this raises an error while it shouldn't:
```php
public function foo($qux): void
{
    $bar = function ($baz) { return $baz + $qux; }
}
```

**Comment by ZhukV** [comment]
Clone is #119

**Comment by podhy** [comment]
same error even if using latest master branch

**Comment by xalopp** [comment]
Also get an error 
```
 104 | ERROR | Use return null; when a function explicitly returns null values and use return; when the function returns void values (Symfony.Functions.ReturnType.Invalid)
```
with Version 3.4.0

for this piece of code:

```
$expected = array_map(
            function (array $model): Product {
                return new Product($model); 
            },
            $input
        );
```
